capstone/src/main/webapp/script.js

```diff
@@ -49,3 +49,137 @@ function  formatCardInHTML(studySet) {
       </div>
     </a>`;
 }
+
+function addCard() {
+  var form = document.getElementById("card_form");
+
+  var newFrontCard = document.createElement("div");
+  newFrontCard.setAttribute("class", "input-field col s6");
+
+  var frontCard = document.createElement("input");
+  frontCard.setAttribute("id", "front");
+  frontCard.setAttribute("type", "text");
+  frontCard.setAttribute("class", "validate input-card");
+
+  var frontLabel = document.createElement("label");
+  frontLabel.setAttribute("for", "front");
+  var frontText = document.createTextNode("Front");
+  frontLabel.append(frontText);
+
+  newFrontCard.append(frontCard);
+  newFrontCard.append(frontLabel);
+
+  form.appendChild(newFrontCard);
+
+  var newBackCard = document.createElement("div");
+  newBackCard.setAttribute("class", "input-field col s6");
+
+  var backCard = document.createElement("input");
+  backCard.setAttribute("id", "back");
+  backCard.setAttribute("type", "text");
+  backCard.setAttribute("class", "validate input-card");
+
+  var backLabel = document.createElement("label");
+  backLabel.setAttribute("for", "back");
+  var backText = document.createTextNode("Back");
+  backLabel.append(backText);
+
+  newBackCard.append(backCard);
+  newBackCard.append(backLabel);
+
+  form.appendChild(newBackCard);
+}
+
+function addStudySet() {
+  var elements = document.getElementsByClassName("input-card");
+  var cards = [];
+
+  for (var i = 0; i < elements.length - 1; i = i + 2) {
+    var contacts = { front: elements[i].value, back: elements[i + 1].value };
+    cards.push(contacts);
+  }
+
+  var user_id = document.getElementById("user_id").value;
```

Comment:
JS variable names should use camelCase: https://google.github.io/styleguide/jsguide.html#naming-local-variable-names.